flytekit/extras/accelerators.py

```diff
@@ -1,3 +1,49 @@
+"""
+Use specific accelerators
+==========================
+
+.. tags:: MachineLearning, Advanced, Hardware
+
+Flyte tasks are very powerful and allow you to select `gpu` resources for your task. However, there are some cases where
```

Comment:
Suggested edit for this section below. (The suggestion tool won't let me leave a suggestion for more than one line, otherwise I'd use that.)

```
Flyte allows you to specify `gpu` resources for a given task. However, in some cases, you may want to use a different accelerator type, such as TPU, specific variations of GPUs, or fractional GPUs. You can configure the Flyte backend to use your preferred accelerators, and those who write workflow code can import the `flytekit.extras.accelerators` module to specify an accelerator in the task decorator.

The `flytekit.extras.accelerators` module provides some constants for known accelerators, listed below, but this is not a complete list. If you know the name of the accelerator, you can pass the string name to the task.
```

Reply:
done